internal/pkg/resource/resource.go

```diff
@@ -19,6 +19,7 @@ const (
 	FlinkRegion           = "Flink region"
 	FlinkIamBinding       = "Flink IAM binding"
 	FlinkStatement        = "Flink SQL statement"
+	GroupMapping          = "group mapping"
```

Comment:
Should the output include "SSO"?
```suggestion
	SsoGroupMapping          = "SSO group mapping"
```